src/gui/ui/hexlineedit.cpp

```diff
@@ -48,7 +48,7 @@ HexLineEdit::HexLineEdit(
     set_value(0);
 }
 
-void HexLineEdit::set_value(uint32_t value) {
+void HexLineEdit::set_value(uint64_t value) {
     QString s, t = "";
     last_set = value;
     s = QString::number(value, base);
```

Comment:
I am fine with storing 64bit values here, but we should make more a 32 bit friendly visualization.